### PR TITLE
Fix ESM build issues with TSC

### PR DESCRIPTION
**Pull request by Mrtenz**
Temporary alternative for #144, which doesn't require bumping the Node.js version.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,5 +9,9 @@
   },
   "include": [
     "./**/*.ts"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist"
```
Comment (FrederikBolding) [inline tsconfig.json:15]: Why is this needed?

Comment (Mrtenz) [reply tsconfig.json:15]: Otherwise TypeScript will try to build `dist` again, but that results in errors since it's also writing to `dist`.

Comment (Mrtenz) [reply tsconfig.json:15]: It's not really needed now that I've added `rimraf` I guess, but it doesn't hurt to add it.

File: tsconfig.esm.json [hunk 2]
```diff
@@ -10,5 +10,9 @@
   },
   "include": [
     "./**/*.ts"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist"
```
Comment (FrederikBolding) [inline tsconfig.esm.json:16]:
```suggestion
    "dist",
    "**/*.test.ts"
```

File: tsconfig.json (hunk 1 above)
Comment (FrederikBolding) [inline tsconfig.json:15]:
```suggestion
    "dist",
    "**/*.test.ts"
```

**Review by FrederikBolding — APPROVED** [review]
